### PR TITLE
[NETBEANS-1074] Module Review web.beans

### PR DESCRIPTION
**Pull request by gilbertoca**
Checked the rat-report.txt file 
Change the license header of Bundle.properties
Add the licenseinfo.xml (little afraid what to put here)
Change the license header of test files

### CODE REVIEW

File: web.beans/licenseinfo.xml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+        <comment type="TEMPLATE_MINIMAL_IP"/>
+    </fileset>
+    <fileset>
+        <file>test/unit/src/org/netbeans/modules/web/beans/xdm/model/alternatives-beans-orig.xml</file>
+        <file>test/unit/src/org/netbeans/modules/web/beans/xdm/model/alternatives-beans.xml</file>
+        <file>test/unit/src/org/netbeans/modules/web/beans/xdm/model/beans-orig.xml</file>
+        <file>test/unit/src/org/netbeans/modules/web/beans/xdm/model/beans.xml</file>
+        <file>test/unit/src/org/netbeans/modules/web/beans/xdm/model/empty-beans.xml</file>
+        <license ref="Apache-2.0-ASF" />
+        <comment type="TEMPLATE_MINIMAL_IP"/>
```
Comment (junichi11) [inline web.beans/licenseinfo.xml]: I'm not sure whether this is correct.
@matthiasblaesing Is this OK?

File: web.beans/src/org/netbeans/modules/web/beans/impl/model/Bundle.properties [hunk 2]
```diff
@@ -14,6 +14,7 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
+
```
Comment (junichi11) [inline web.beans/src/org/netbeans/modules/web/beans/impl/model/Bundle.properties]: This is not necessary.

File: web.beans/src/org/netbeans/modules/web/beans/resources/Bundle.properties [hunk 3]
```diff
@@ -14,6 +14,7 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
+
```
Comment (junichi11) [inline web.beans/src/org/netbeans/modules/web/beans/resources/Bundle.properties]: This is not necessary.

File: web.beans/licenseinfo.xml [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+        <file>src/org/netbeans/modules/web/beans/resources/event.png</file>
+        <file>src/org/netbeans/modules/web/beans/resources/observer.png</file>
+        <license ref="Apache-2.0-ASF" />
+        <comment type="COMMENT_UNSUPPORTED" />
+    </fileset>
+    <fileset>
+        <file>src/org/netbeans/modules/web/beans/resources/Interceptor.template</file>
+        <file>src/org/netbeans/modules/web/beans/resources/Qualifier.template</file>
+        <file>src/org/netbeans/modules/web/beans/resources/Scope.template</file>
+        <file>src/org/netbeans/modules/web/beans/resources/Stereotype.template </file>
```
Comment (junichi11) [inline web.beans/licenseinfo.xml]: Please remove the last whitespace.
```
BUILD FAILED
/home/travis/build/apache/incubator-netbeans/nbbuild/build.xml:1530: Failed VerifyLibsAndLicenses test(s):
Some licenseinfo.xml files failed verification:
web.beans/licenseinfo.xml referenced file not found 'src/org/netbeans/modules/web/beans/resources/Stereotype.template '
```

File: web.beans/src/org/netbeans/modules/web/beans/impl/model/Bundle.properties (hunk 2 above)
Comment (gilbertoca) [reply web.beans/src/org/netbeans/modules/web/beans/impl/model/Bundle.properties]: Just trying to be coherent - some file has other not.

File: web.beans/src/org/netbeans/modules/web/beans/resources/Bundle.properties (hunk 3 above)
Comment (gilbertoca) [reply web.beans/src/org/netbeans/modules/web/beans/resources/Bundle.properties]: Just trying to be coherent - some file has other not. What do you suggest?

Comment (junichi11) [reply web.beans/src/org/netbeans/modules/web/beans/resources/Bundle.properties]: It is not related to relicensing. So you should not change this file.

File: web.beans/licenseinfo.xml (hunk 1 above)
Comment (gilbertoca) [reply web.beans/licenseinfo.xml]: That's a problem, because this module depends on several other ones. I will remove that part form the lisenceinfo file.

Comment (matthiasblaesing) [reply web.beans/licenseinfo.xml]: I do this:
- build the full netbeans cluster - in the root of the checkout run `ant -Dcluster.config=full` this takes about 15 minutes on my notebook
- open the module then in the IDE and check the project properties, make sure a JDK 8 is select in Libraries -> Java Platform
- The you can run the unittests via "Test" from the projects context menu

Comment (gilbertoca) [reply web.beans/licenseinfo.xml]: It took 50 minutes + 10 minutes with test (old notebook). Have two test methods failed before and after. I will let someone with more experience verify the test after merge.

File: web.beans/src/org/netbeans/modules/web/beans/impl/model/Bundle.properties (hunk 2 above)
Comment (junichi11) [reply web.beans/src/org/netbeans/modules/web/beans/impl/model/Bundle.properties]: Please revert this as well.

File: web.beans/licenseinfo.xml (hunk 1 above)
Comment (junichi11) [reply web.beans/licenseinfo.xml]: Thanks for your work! I also verified that the two test methods failed. Please write about that in the confluence page. If you cannot edit it, please ask @geertjanw .

**Comment by junichi11** [comment]
If above change is reverted, we should "squash and merge".

**Comment by gilbertoca** [comment]
@geertjanw  I verified that two test methods failed. I don't have write permission to inform that in the confluence page. Can you do that for us?

**Comment by geertjanw** [comment]
Gilberto Caetano de Andrade (gilbertoca) has permission now. :-)

**Comment by gilbertoca** [comment]
Thank you guys (@geertjanw @junichi11 @matthiasblaesing ), I've update the confluence page.

File: web.beans/src/org/netbeans/modules/web/beans/impl/model/Bundle.properties (hunk 2 above)
Comment (matthiasblaesing) [reply web.beans/src/org/netbeans/modules/web/beans/impl/model/Bundle.properties]: @gilbertoca could you please address @junichi11 comment? This is not really a deal breaker, but we should try to not introduce unnessary changes. Thank you!

Comment (gilbertoca) [reply web.beans/src/org/netbeans/modules/web/beans/impl/model/Bundle.properties]: I've already done at the same day - See the [Revert change](https://github.com/apache/incubator-netbeans/pull/694/commits/25cc426810c520e64afb3e58d3beeb70dbc1fd61) commit

Comment (matthiasblaesing) [reply web.beans/src/org/netbeans/modules/web/beans/impl/model/Bundle.properties]: The linked revert covers:
`web.beans/src/org/netbeans/modules/web/beans/resources/Bundle.properties`
this file is:
`web.beans/src/org/netbeans/modules/web/beans/impl/model/Bundle.properties`
maybe you overlooked to commit the second file?

Comment (gilbertoca) [reply web.beans/src/org/netbeans/modules/web/beans/impl/model/Bundle.properties]: I'm sorry, I will revert it as well

**Pull request merged**